index.html

```diff
@@ -8,11 +8,22 @@
 <body>
     <p id="demo"></p>
     <p id="demo1"></p>
+    <p id="demo2"></p>
+
     <script src="executeCommand.js"></script>
     <script>
-        const numbers = [2,3,1,5,4]
-       
-        sortedGivenNumber(numbers);
```

Comment:
line 13 and 15 should not want to delete from the code. it should want a first script tag. because it is the our first task. which is sorted the given number.